### DISCUSSION TITLE
Which is official release - Github or Sourceforge?

### DISCUSSION THREAD

**Discussion by amishmm** [webmin/webmin#2076] [Q&A] [answered]
Hello

Can webmin developers please clarify if releases on Github are official or the one on sourceforge?

Github release page - https://github.com/webmin/webmin/releases
Sourceforge files page - https://sourceforge.net/projects/webadmin/files/webmin/

There was similar discussion in 2019 where sourceforge was mentioned as official one:
https://sourceforge.net/p/webadmin/mailman/webadmin-list/thread/94f498c3-e79e-1347-aae1-923f6bcb1d85%40gmail.com/#msg36663808

But now it seems that tar.gz files on github and sourceforge both are now same.

Has developers migrated completely to github?

Please confirm.

Thank you.

**Comment by iliaross** [comment, marked as answer]
> Can webmin developers please clarify if releases on Github are official or the one on sourceforge?

Development happens in GitHub. Releases are posted to both GitHub and SourceForge. Both are official.

**Reply by amishmm** [reply]
Is same true for usermin too? i.e. both github release and sourceforge release are official?

**Reply by iliaross** [reply]
Yes, the same is true for Usermin.
